### ISSUE TITLE
WCAG issue (1.4.3 Contrast (Minimum))

### ISSUE THREAD

**Issue by sorensensig** [digdir/baksia#921]
List of issues related to 1.4.3 Contrast (Minimum) found during WCAG walkthrough of Baksia:

```[tasklist]
### General issues
- [ ] The "fjern" button and "søk" placeholder in the search bar fails contrast requirements with 1.45:1 (req: 4.5:1)
- [ ] The search icon in the search bar fails contrast requirements with 2.05:1 (req: 3:1)
```
